features/input-range.yml

```
@@ -0,0 +1,16 @@
+name: '<input type="range">'
+description: The `<input>` element allows the user to select a value using a slider widget.
```

(pet peeve: select is one of those deeply overloaded words when talking about UI. Going abstract here helps a little.)

```suggestion
description: The `<input type="range">` element allows the user to choose a value using a slider widget.
```